File: 1° Período - Introdução a Programação 1/Exercicios em Python/Lista 5 - Python/q1.py
#Faça um algoritmo que imprima todos os números pares compreendidos entre 95 e 907. O
#algoritmo deve também calcular e mostrar a soma destes valores.

soma=0
for i in range(95, 907):
    if i%2==0:
        soma=soma+i
        print(i)
print("SOMA: ", soma)

File: 1° Período - Introdução a Programação 1/Exercicios em C/Lista 5 - C/q1.c
//Faça um algoritmo que imprima todos os números pares compreendidos entre 85 e 907. O
//algoritmo deve também calcular e mostrar a soma destes valores.

#include<stdio.h>
int main(){
	int i;
	for(i=85; i<=600; i++){
		if(i%2==0){
			printf("%d\n", i);
		}
	}
}
